echo/frontend/src/hooks/useCopyQuote.ts

```
@@ -23,70 +23,83 @@ export const useCopyQuote = () => {
 
     // Build the formatted quote with context
     stringBuilder.push(
-      `# Quote from [${quote.conversation_id.participant_name}](${window.location.origin}/${language}/projects/${projectId}/conversation/${quote.conversation_id.id}/transcript)`,
+      `# Quote from [${conversation?.participant_name}](${window.location.origin}/${language}/projects/${projectId}/conversation/${conversation?.id}/transcript)`,
     );
-    stringBuilder.push(`"${quote.text}"`);
+    stringBuilder.push(`"${quote.description}"`);
+    stringBuilder.push(`${quote.verbatim_transcript}`);
+
+    try {
+    const startIndex = parseInt(quote.relevant_index?.split(":")[0] ?? "0");
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)